data/map_events.s

```diff
@@ -14,4 +14,5 @@
 
     .section .rodata
 
+    .align 2
```

Comment:
I believe this is redundant (it would be immediately followed by the `.align 2` in the first map's `events.inc`)

Reply:
Fair enough, I'll remove it.